index.html

```diff
@@ -3409,6 +3409,44 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+      <section>
+        <h3>Media Types</h3>
+
+        <p>
+          Media types as defined in [[RFC6838]] serve a useful purpose with verifiable 
+          credentials, especially in regards to identifcation of a particular syntax 
+          that is in use with a verifiable credential.
+        </p>
+        <p>
+          Syntaxes SHOULD be identified by a media type, and certain conventions as 
+          outlined in this section should be followed when defining or using media types 
+          with verifiable credentials.
+        </p>
+        <p>
+          Any media types associated with the core data model are listed in the section on
+          <a href="#iana-considerations">IANA Considerations</a> for registration with IANA.
+        </p>
+        <p>
+          When defining a media type for use with verifiable credentials, for instance in a 
+          specification that defines a specific syntax, use of the term `verifiable` implies
+          the presence of a <a href="#proofs-signatures">proof</a> with the credential.  
```

Comment:
Yikes to this... I might defer this language to a separate PR to avoid blocking this valuable other changes.

`proof` is NOT required to make a `VerifiableCredential`...  asserting that `proof` makes a `VerifiableCredential` implies data integrity proofs as a defacto standard representation / serialization for the core data model.

Reply:
yeah - this is one reason why i wanted this as a draft first - we do i think want to note that some cryptographic proof is there, not necessarily that it is in the credential as an embedded proof (as it often is not) but that it should be present according to the securing method in use

Reply:
Can we remove this paragraph on "proof" ? and move it to an issue to discuss?

Reply:
#1060 opened, paragraph removed to refine language